greenlight compilation error

Trying to debug my green wallet with lightning and I would like to use the sdk-cli. However, I get the following compilation error 

```
➜  sdk-cli git:(main) ✗ cargo run
   Compiling gl-client v0.2.0 (https://github.com/Blockstream/greenlight.git?rev=e38a37613da7558c853f24be700c193f194a6bc9#e38a3761)
error[E0412]: cannot find type `DelexpiredinvoiceRequest` in module `cln`
    --> /home/vincent/.cargo/git/checkouts/greenlight-e41cf58dbd255ee0/e38a376/libs/gl-client/src/signer/model/mod.rs:41:28
     |
41   |     DelExpiredInvoice(cln::DelexpiredinvoiceRequest),
     |                            ^^^^^^^^^^^^^^^^^^^^^^^^ help: a struct with a similar name exists: `DelinvoiceRequest`
     |
    ::: /home/vincent/github/work/breez-sdk/tools/sdk-cli/target/debug/build/cln-grpc-7a25774ebd2ceb74/out/cln.rs:1800:1
     |
1800 | pub struct DelinvoiceRequest {
     | ---------------------------- similarly named struct `DelinvoiceRequest` defined here

error[E0433]: failed to resolve: use of undeclared type `DelexpiredinvoiceRequest`
  --> /home/vincent/.cargo/git/checkouts/greenlight-e41cf58dbd255ee0/e38a376/libs/gl-client/src/signer/model/cln.rs:27:62
   |
27 |     "/cln.Node/DelExpiredInvoice" => Request::DelExpiredInvoice(DelexpiredinvoiceRequest::decode(p)?),
   |                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^
   |                                                                 |
   |                                                                 use of undeclared type `DelexpiredinvoiceRequest`
   |                                                                 help: a struct with a similar name exists: `DelinvoiceRequest`

error[E0599]: no method named `bolt11` found for reference `&cln_grpc::pb::PreapproveinvoiceRequest` in the current scope
   --> /home/vincent/.cargo/git/checkouts/greenlight-e41cf58dbd255ee0/e38a376/libs/gl-client/src/signer/resolve.rs:102:40
    |
102 |                     l.invstring.0 == r.bolt11().as_bytes()
    |                                        ^^^^^^-- help: remove the arguments
    |                                        |
    |                                        field, not a method

Some errors have detailed explanations: E0412, E0433, E0599.
For more information about an error, try `rustc --explain E0412`.
The following warnings were emitted during compilation:

warning: gl-client@0.2.0: Using default NOBODY cert.
warning: gl-client@0.2.0: Set "GL_CUSTOM_NOBODY_KEY" and "GL_CUSTOM_NOBODY_CERT" to use a custom cert.
```

Probably is it a greenlight issue but idk how it is import inside the SDK-cli, so I will start reporting the problem from here

--- a/libs/sdk-core/Cargo.toml
+++ b/libs/sdk-core/Cargo.toml
@@ -16,7 +16,7 @@ hex = { workspace = true }
 # The switch to 0.2 will happen with https://github.com/breez/breez-sdk/pull/724
 gl-client = { git = "https://github.com/Blockstream/greenlight.git", features = [
     "permissive",
-], rev = "e38a37613da7558c853f24be700c193f194a6bc9" }
+], rev = "16fa13d40f4fc271cf99febf3f40246b6b23716a" }
 zbase32 = "0.1.2"
 base64 = { workspace = true }
 chrono = "0.4"

